cx-server-companion/life-cycle-scripts/cx-server

```diff
@@ -27,6 +27,8 @@ fi
         --env DEVELOPER_MODE \
         --env host_os=unix \
         --env cx_server_path="${PWD}" \
+	--env PPIPER_INFRA_IT_CF_PASSWORD \
```

Comment:
would be in favor of a more generic approach, e.g. configure somewhere a list of environment variables

Reply:
As @radsoulbeard said: As it is productive code and the content of this file is shown to the user I would like to see another approach which is nearer to productive uses cases (e.g. using config as code and configuring credentials productively)